### ISSUE TITLE
Grammatical mistake in playback sounds for rpt_telemetry.c

### ISSUE THREAD

**Issue by encbar5** [AllStarLink/app_rpt#142]
The handle_varcmd_tele funtion in rpt_telemetry.c, around line 638 is using the sound file for digits/2 in order to say "connected 2" meaning "connected to" in English. This does not translate into other languages. This needs to be changed to a sound file that actually correctly uses the preposition "to" so that translations can be provided.
I would have made this change myself, but I could not find a gsm file for this word in the Allison recordings. Maybe someone else knows more about the sound files?
Edit: There are two occurrences of this in the file, second one around line 1656.

**Comment by Allan-N** [comment]
It looks like rpt_telemetry.c has been updated to use "rpt/connected-to" and our packaging includes `connected-to.gsm`.

It looks like this issue has been resolved.

**Comment by InterLinked1** [comment]
> It looks like rpt_telemetry.c has been updated to use "rpt/connected-to" and our packaging includes `connected-to.gsm`.
> 
> It looks like this issue has been resolved.

Yes, looks like the linked PR was merged but the commit message wasn't using the markup to close the issue, so it didn't autoclose.
